build/devops_gated.yml

```diff
@@ -253,7 +253,6 @@ jobs:
     - Cmd
     - msbuild
     - visualstudio
-  condition: false
```

Comment:
>condition: false [](start = 2, length = 16)

Is there a reason to enable only this job?

Reply:
I would enable all, but we don't have a CMakeLists yet. I enabled this one to validate that the c build tools building of the traceability tool works.